_data/links.yml

```
@@ -200,11 +200,11 @@
     - 'background-position-x: right'
 
   - name: Rosetta Stone
-    href: https://he2.tellmemorecampus.com/
+    href: https://login.rosettastone.com/
     class: tile-small-wide
-    background_image: https://resources.rosettastone.com/assets/ss/221270--8920100/App_Themes/Default/Global/0101UnidentifiedHomePage/Images/r/logo_RS.png
+    background_image: https://upload.wikimedia.org/wikipedia/en/thumb/6/67/Rosetta_Stone_logo.svg/1920px-Rosetta_Stone_logo.svg.png
```

No need for an oversized logo here.
```suggestion
    background_image: https://upload.wikimedia.org/wikipedia/en/thumb/6/67/Rosetta_Stone_logo.svg/320px-Rosetta_Stone_logo.svg.png
```
This reduces file size from 151 KB to 20 KB.